packages/intrepid2/unit-test/Shared/ArrayTools/Cuda/test_02.cpp

```diff
@@ -54,7 +54,8 @@ int main(int argc, char *argv[]) {
   const bool verbose = (argc-1) > 0;
   Kokkos::initialize();
 
-  const int r_val = Intrepid2::Test::ArrayTools_Test02<double,Kokkos::Cuda>(verbose);
+  const int r_val = Intrepid2::Test::ArrayTools_Test02
+    <double,Kokkos::Cuda/* Kokkos::Device<Kokkos::Cuda,Kokkos::CudaSpace>*/ >(verbose);
```

Comment:
@kyungjoo-kim, why do you do something different here than in Cuda/test_01.cpp?  Is it possible to be consistent with that as well as test_03.cpp?

Reply:
I planned to work all three tests and there are so many things to fix. I was tired of changing things. So, I just fix test 01. I am going to separate this work into multiple PRs. Once I found a pattern, I may ask you and mauro to help convert the tests.

Reply:
Got it.  Would you please add a comment here indicating that the commented-out bit is what we intend to do once test_02 is fixed?